portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -66,4 +88,17 @@ public void createEntitys(){
     datastore.put(commentEntity);
   }
 
+  public void populateList(PreparedQuery results){
+    this.comments = new ArrayList<>();
+    for (Entity commentEntity : results.asIterable()) {
+      this.idGiven = commentEntity.getKey().getId();
```

Comment:
Why do you need to assign these to instance variables? Why not just pass them directly to the comment constructor?